fe1-web/src/features/evoting/network/ElectionHandler.ts

```diff
@@ -64,6 +66,40 @@ export function handleElectionSetupMessage(msg: ProcessableMessage): boolean {
   return true;
 }
 
+/**
+ * Handles an ElectionOpen message by opening the election.
+ *
+ * @param msg - The extended message for opening an election
+ */
+export function handleElectionOpenMessage(msg: ProcessableMessage) {
+  console.log('Handling Election open message');
+  if (
+    msg.messageData.object !== ObjectType.ELECTION ||
+    msg.messageData.action !== ActionType.OPEN
+  ) {
+    console.warn('handleElectionOpenMessage was called to process an unsupported message', msg);
+    return false;
+  }
+  const makeErr = (err: string) => `election/open was not processed: ${err}`;
+  const storeState = getStore().getState();
+  const lao = getCurrentLao(storeState);
+  if (!lao) {
+    console.warn(makeErr(STRINGS.no_active_lao));
+    return false;
+  }
+  const ElectionOpenMsg = msg.messageData as OpenElection;
+  const election = getEventFromId(storeState, ElectionOpenMsg.election) as Election;
+  if (!election) {
+    console.warn(makeErr('No active election to end'));
+    return false;
+  }
+
+  // Change election status here such that it will change the election display in the event list
+  election.electionStatus = ElectionStatus.RUNNING;
+  dispatch(updateEvent(lao.id, election.toState()));
```

Comment:
```suggestion
  dispatch(updateEvent(msg.laoId, election.toState()));
```

Reply:
👍 Should do the same for `handleCastVoteMessage`, `handleElectionEndMessage`, `handleElectionResultMessage` in a future PR and also add tests for them!